package.json

```diff
@@ -27,23 +27,27 @@
   "matrix-react-parent": "matrix-react-sdk",
   "scripts": {
     "reskindex": "reskindex -h src/header",
-    "build:emojione": "cpx \"node_modules/emojione/assets/svg/*\" vector/emojione/svg/",
+    "build:res": "cpx \"{src/skins/vector/fonts,src/skins/vector/img}/**\" webapp/ && cpx \"{res/media,res/vector-icons}/**\" webapp/",
+    "build:config": "cpx config.json webapp/",
```

Comment:
I was going to keep it in the root directory so it's more obvious to find: I don't really want a `vector` directory hanging around for the sole purpose of containing the config: people will have to move the config file but I don't think that's a big deal.